chia/full_node/full_node.py

```diff
@@ -89,6 +89,14 @@ class PeakPostProcessingResult:
     lookup_coin_ids: List[bytes32]  # The coin IDs that we need to look up to notify wallets of changes
 
 
+@dataclasses.dataclass(frozen=True)
+class WalletUpdate:
```

Comment:
I think it would be good to add a comment describing what this class is used for. It's not obvious from its name.

does `fork_height` refer to a reorg? The name suggests it does, but I wouldn't expect to find that field in here.

Reply:
If there was a reorg, yes, it tells the wallet about the reorg but usually its just from `peak_height - 1` to `peak_height`. Its how the protocol works, so why wouldn't you expect it to be in here? The `WalletUpdate` class is simply a dataclass container to bundle some items to put them into a queue.